ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -768,15 +775,25 @@ private KsqlConfig(final ConfigGeneration generation,
     for (final ConfigValue config : ksqlStreamConfigProps.values()) {
       props.put(config.key, config.value);
     }
+
+    props.putAll(getConfigsForPrefix(REPORTER_CONFIGS_PREFIXES));
     return Collections.unmodifiableMap(props);
   }
 
   public Map<String, Object> getKsqlAdminClientConfigProps() {
-    return getConfigsFor(AdminClientConfig.configNames());
+    final Map<String, Object> map = new HashMap<>();
```

Comment:
If you want ksql cluster id and application propagate to AdminClient, you should add as part of admin client properties:
metrics.context.cluster.id, metrics.context.ksql.application.id

Reply:
I updated it so that `metrics.context.resource.cluster.id=<ksql-service-id>` is being passed in for adminClient, producer, and streams